templates/fine-tune-llm_v2/cookbooks/continue_from_checkpoint/README.md

```diff
@@ -0,0 +1,57 @@
+# Continue fine-tuning from a previous checkpointing
+
+**⏱️ Time to complete**: 40 minutes
+
+This document assumes that you have familiarized yourself with the main fine-tuning guide of this template.
+In this folder of the template, we showcase how a checkpoint that was created earlier can be used to start a training from.
+We case use this, for example, if we think that starting from a given checkpoint will give us a performance advantage.
+
+There are two types of checkpoints to considere here: Full-parameter checkpoints, and LoRA-adapter checkpoints.
+For starters, we advise against combining the two (by training a LoRA adapter ontop of a full-parameter checkpoint), because serving the resulting LoRA adapter will require the full-parameter checkpoint.
+
+## How to fine-tune from a previous checkpointing
+
+
+```python
+# [Optional] You can set the WandB API key to track model performance
+# !export WANDB_API_KEY={YOUR_WANDB_API_KEY}
+
+# Continue LoRA fine-tuning on the GSM8k dataset with Llama 3 8B
+!llmforge dev finetune llama-3-8b.yaml
+```
+
+## What we are fine-tuning on
+
+Running the above command will fine-tune on the [GSM8k dataset](https://huggingface.co/datasets/gsm8k). 
```

Comment:
In here show some wandb charts that shows that the continuation of finetuning from the second half is actually better than starting from scratch.